src/controllers/userProfile.js

```diff
@@ -13,7 +16,7 @@ export default class changeUserProfile {
 
   static changeMyProfileInfo(request, response) {
     request.body.birthDay = validateDate(request.body.birthDay);
-    request.body.profileImage = (typeof request.file === 'undefined') ? null : `http://localhost:4000/${request.file.path}`;
+    request.body.profileImage = (typeof request.file === 'undefined') ? null : `${process.env.URL}/${request.file.path}`;
```

Comment:
@EugeneMunya  Please change the  name of the variable from URL to LOCAL_HOST

Reply:
I have corrected that mistake for now you can check